EIPS/eip-1.md

```diff
@@ -67,54 +87,88 @@ These calls generally result in a "rough consensus" around what EIPs should be i
+This is the final review window for an EIP before moving to `Final`. An EIP editor will assign `Last Call` status and set a review end date (`last-call-deadline`), typically 14 days later.
+  - If this period results in necessary normative changes it will revert the EIP to `Review`.
+
+#### Final
+
+This EIP represents the final standard. A Final EIP exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.
 
 A PR moving an EIP from Last Call to Final SHOULD contain no changes other than the status update. Any content or editorial proposed change SHOULD be separate from this status-updating PR and committed prior to it.
 
-**Stagnant** - Any EIP in `Draft` or `Review` or `Last Call` if inactive for a period of 6 months or greater is moved to `Stagnant`. An EIP may be resurrected from this state by Authors or EIP Editors through moving it back to `Draft` or it's earlier status. If not resurrected, a proposal may stay forever in this status.
+#### Stagnant
+
+Any EIP in `Draft` or `Review` or `Last Call` if inactive for a period of 6 months or greater is moved to `Stagnant`. An EIP may be resurrected from this state by Authors or EIP Editors through moving it back to `Draft` or it's earlier status. If not resurrected, a proposal may stay forever in this status.
+
+*EIP Authors are notified of any algorithmic change to the status of their EIP*
 
->*EIP Authors are notified of any algorithmic change to the status of their EIP*
+#### Withdrawn
 
-**Withdrawn** - The EIP Author(s) have withdrawn the proposed EIP. This state has finality and can no longer be resurrected using this EIP number. If the idea is pursued at later date it is considered a new proposal.
+The EIP Author(s) have withdrawn the proposed EIP. This state has finality and can no longer be resurrected using this EIP number. If the idea is pursued at later date it is considered a new proposal.
 
-**Living** - A special status for EIPs that are designed to be continually updated and not reach a state of finality. This includes most notably EIP-1.
+#### Living
+
+A special status for EIPs that are designed to be continually updated and not reach a state of finality. This includes most notably EIP-1.
+
+### Technical Assistants
+
+Core EIPs are intensely scrutinized by the client teams and other developers for technical soundness, but the EIP Editors may still lack relevant expertise. And the numerous ERCs cover a wide and growing range of applications that are even more likely to be outside the expertise of any Editor.  Historically, this has put a strain on the Editors' responsibility to maintain quality.  Therefore, the Editors *at their discretion* may require Technical Assistants with the relevant expertise who can review the Author's Idea and work with the Author to ensure a technically sound proposal.
```

Comment:
This is a good idea, but I think we've tried on a few occasions to activate these people, but have failed.

Reply:
@xinbenlv think we need make it a solid policy and try again. We've also failed to get enough new Editors to handle all of the proposals we get.  I think part of the difficulty is that proposals span too wide of a range of technical expertise for a small group of Editors.  Many publishers handle this by weeding out articles via peer review before editing even starts.

Reply:
some kind of nominal honorarium or [on-chain?] badge of honor for participating never hurts to make folks feel appreciated for this "unpaid overtime".  i also think there's a cultural change where having engineers from your *competitors*  reviewing and approving your EIPs could really be a prestige/maturity symbol! in any case, I am a huge +1 to formalizing recognition of review and getting external/horizontal/non-editor review.

Reply:
Expecting laurels for service to Ethereum?!

Reply:
> This is a good idea, but I think we've tried on a few occasions to activate these people, but have failed.

I was not paying attention when these occasions came and went, does anyone have links to catch me up on these attempts?